tests/python/pants_test/java/test_nailgun_protocol.py

```diff
@@ -205,6 +205,43 @@ def test_isatty_from_env_mixed(self):
       (False, True, False)
     )
 
+  def _make_mock_stream(self, isatty, fileno):
+    mock_stream = mock.Mock()
+    mock_stream.isatty.return_value = isatty
+    mock_stream.fileno.return_value = fileno
+    return mock_stream
+
+  @mock.patch('os.ttyname', autospec=True, spec_set=True)
+  def test_isatty_to_env_with_mock_tty(self, mock_ttyname):
+    mock_ttyname.return_value = '/dev/ttys000'
+    mock_stdin = self._make_mock_stream(True, 0)
+    mock_stdout = self._make_mock_stream(True, 1)
+    mock_stderr = self._make_mock_stream(True, 2)
+
+    self.assertEquals(
+      NailgunProtocol.isatty_to_env(mock_stdin, mock_stdout, mock_stderr),
+      {
+        'NAILGUN_TTY_0': b'\x01',
+        'NAILGUN_TTY_1': b'\x01',
+        'NAILGUN_TTY_2': b'\x01',
+        'NAILGUN_TTY_PATH_0': '/dev/ttys000',
+        'NAILGUN_TTY_PATH_1': '/dev/ttys000',
+        'NAILGUN_TTY_PATH_2': '/dev/ttys000',
+      })
+
+  def test_isatty_to_env_without_tty(self):
+    mock_stdin = self._make_mock_stream(False, 0)
+    mock_stdout = self._make_mock_stream(False, 1)
+    mock_stderr = self._make_mock_stream(False, 2)
+
+    self.assertEquals(
+      NailgunProtocol.isatty_to_env(mock_stdin, mock_stdout, mock_stderr),
+      {
+        'NAILGUN_TTY_0': b'\x00',
+        'NAILGUN_TTY_1': b'\x00',
+        'NAILGUN_TTY_2': b'\x00',
```

Comment:
the expected values should all be `b'0'` in this test case.

Reply:
Fixed in 9d221e6.